taxi/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.db.models.constraints import UniqueConstraint
+
+
+class Manufacturer(models.Model):
+    name = models.TextField()
+    country = models.TextField()
+
+    def __str__(self):
+        return f"{self.name} from {self.country}"
+
+
+class Car(models.Model):
+    model = models.TextField()
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.CASCADE,
+    )
+

```

Comment:
where are field drivers?
pls consider the scheme again:
https://user-images.githubusercontent.com/80070761/159295912-d02c7080-09a7-41ec-aa86-b0ae3afdd75b.png